scripts/install_third_party.sh

```diff
@@ -123,12 +123,14 @@ if [ ! -d "$TOOLS_DIR/numpy-1.6.1" ]; then
       echo "  Numpy is required for Oppia to operate properly, but pip wasn't found"
       echo "  on your local machine."
       echo ""
-      echo "  Please see \"Installing Oppia\" on the Oppia developers' wiki page:"
+      echo "  Please see \"Installing Oppia\" on the Oppia developers' wiki page:"      
 
       if [ "${OS}" == "Darwin" ] ; then
         echo "    https://github.com/oppia/oppia/wiki/Installing-Oppia-%28Mac-OS%29"
+        echo " You can also optionally install pip by using the install-pip.sh script."
```

Comment:
Actually let's remove this, I'll add a similar statement to both wiki pages instead.